packages/bruno-api-typescript/src/generator/apiDefinitionGenerator.ts

```
@@ -54,15 +68,87 @@ function extractPathParams(url: string): string[] {
   return params;
 }
 
+function extractPathParamTokens(url: string): string[] {
+  const params: string[] = [];
+  let processedUrl = url.replace(/\{\{URL\}\}/g, '');
+
+  const brunoVarPattern = /\{\{([^}]+)\}\}/g;
+  let match;
```

Biome (2.4.15)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)